ansible/tasks/test-image.yml

```diff
@@ -52,11 +52,11 @@
   when: ebssurrogate_mode
 
 - name: Drop pgTap extension
-  shell: /usr/lib/postgresql/bin/psql -U postgres -h localhost -d postgres -c "DROP extension if exists pgtap";
+  shell: /usr/lib/postgresql/bin/psql -U supabase_admin -h localhost -d postgres -c "DROP extension if exists pgtap";
   when: ebssurrogate_mode
 
 - name: Drop extension test function
-  shell: /usr/lib/postgresql/bin/psql -U postgres -h localhost -d postgres -c "DROP FUNCTION IF EXISTS install_available_extensions_and_test";
+  shell: /usr/lib/postgresql/bin/psql -U supabase_admin -h localhost -d postgres -c "DROP FUNCTION IF EXISTS install_available_extensions_and_test";
```

Comment:
This has been fixed; it's no longer necessary to call them using `supabase_admin`

Reply:
Awesome, reverted!